src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -881,32 +1017,80 @@ public bool TryFormat(Span<byte> utf8Destination, out int bytesWritten, [StringS
         /// <summary>Explicitly converts a half-precision floating-point value to its nearest representable <see cref="float" /> value.</summary>
         /// <param name="value">The value to convert.</param>
         /// <returns><paramref name="value" /> converted to its nearest representable <see cref="float" /> value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
As @jkotas wrote in https://github.com/dotnet/runtime/pull/81632#discussion_r1096719739, we force inline only selected methods. Typically only methods that are frequently on a hot path and would not increase the produced assembly size a lot.

Moreover, with recent introduction of dynamic PGO .NET is now capable of profiling itself as it runs, so it's possible that if it makes sense to inline this method, it's going to get inlined by the JIT when promoting code to Tier 1.